src/query/functions/tests/it/aggregates/testdata/agg.txt

```diff
@@ -694,6 +694,16 @@ evaluation (internal):
 +--------+----------------------------------------------------------------------------+
 
 
+ast: stddev_samp(dec)
+evaluation (internal):
```

Comment:
Better to add a test case to window function which in the SQL logic test suites.